### PR TITLE
Improve layout of search form to bring results higher up the page

### PR DESCRIPTION
**Pull request by Guntrisoft**
**Before**
screen shot 2016-11-25 at 16 05 42

**After**
screen shot 2016-11-25 at 16 24 12

- making the form inline, so it fits on one row

### CODE REVIEW

File: app/views/appointments/search.html.erb [hunk 1]
```diff
@@ -2,30 +2,40 @@
...
-              autoUpdateInput: false,
-              locale: {
-                cancelLabel: 'Clear',
-                format: 'DD/MM/YYYY'
+<%= panel do %>
+  <div class="panel-body">
+    <%= form_for @search, method: 'GET', url: { action: :search }, html: { class: 'form-inline' } do |f| %>
+      <div class="form-group">
+        <%= f.label :q, 'Search term' %>
+        <%= f.text_field :q, use_label: false, class: 't-q appointment-search-term form-control', placeholder: 'Ref, Name or Guider' %>
```
Comment (benbarnett) [inline app/views/appointments/search.html.erb]: Can we drop the caps & abbreviation on these: `Reference, name or guider`  should fit?

**Comment by benbarnett** [comment]
Not sure how I feel about the border around the form. What do you reckon?

**Comment by Guntrisoft** [comment]
@benbarnett removed the border

File: app/views/appointments/search.html.erb (hunk 1 above)
Comment (Guntrisoft) [inline app/views/appointments/search.html.erb]: done

**Pull request merged**